crates/core/src/tls.rs

```diff
@@ -109,3 +110,30 @@ impl NativeRootsExt for reqwest::ClientBuilder {
             .fold(self, reqwest::ClientBuilder::add_root_certificate)
     }
 }
+
+pub fn load_certs_from_paths(
+    paths: &[std::path::PathBuf],
+) -> anyhow::Result<Vec<rustls::pki_types::CertificateDer<'static>>> {
+    paths
+        .iter()
+        .map(read_certs_from_path)
+        .flat_map(|result| match result {
+            Ok(vec) => vec.into_iter().map(Ok).collect(),
+            Err(er) => vec![Err(er)],
+        })
+        .collect::<Result<Vec<_>, _>>()
+}
+
+pub fn read_certs_from_path(
+    path: &std::path::PathBuf,
```

Comment:
```suggestion
    path: impl AsRef<Path>,
```

This requires some other changes but should be slightly more flexible this way